transmonee_dashboard/src/transmonee_dashboard/pages/base_page.py

```diff
@@ -731,12 +704,9 @@ def get_disag_total(data, indicator, dimension, default_total="Total"):
 
     data_disag_col = data[data["CODE"] == indicator][dimension]
     max_val_count = data_disag_col.value_counts().idxmax()
-    max_is_total = max_val_count == default_total
```

Comment:
This is my commit: use total if present else get the most used code. Simplifies the code and sort out some problems it used to cause otherwise. Other particularities/non consistent use of sdmx codes in the data should be treated at the etl level.

Reply:
Thanks Alberto... @jamescw , this was the fix to the card displaying wrong values that I previously reported... also to some charts showing wrong data.